tests/pw/README.MD

```
@@ -1,389 +1,574 @@
+**Fix:**
+```bash
+# Run setup again
+npm run docker:setup
+```
 
-<br/>
+### Docker Warning About Dokan Pro
 
-:radio_button: Run all tests
+If you see:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)